### ISSUE TITLE
`gips_mask` applies ACCA automagically if the product is in the project directory

### ISSUE THREAD

**Issue by ircwaves** [Applied-GeoSolutions/gips#40]
Given 
```
gips_mask  --suffix=-agmasked --filemask cdlagmask.tif a_gips_project_dir
```
it seems like only the `cdlagmask.tif` should be applied to the images in that gips project directory -- even if there is a corresponding `acca` product in that gips_project directory.

Having the feature that does both the product and file mask seems nice, but there should at least be a way to turn it off, or better yet, you should have to explicitly specify that you want both the filemask and the product masks applied.

What do you all think?

**Comment by mcorbiere** [comment]
I agree acca (and any mask for that matter) should only be applied if you specifically say so. Especially since bqashadow with Landsat 8 now, we don't want acca being applied as well.
